go.sum

```diff
@@ -215,21 +201,16 @@ github.com/bgentry/go-netrc v0.0.0-20140422174119-9fd32a8b3d3d h1:xDfNPAt8lFiC1U
 github.com/bgentry/go-netrc v0.0.0-20140422174119-9fd32a8b3d3d/go.mod h1:6QX/PXZ00z/TKoufEY6K/a0k6AhaJrQKdFe6OfVXsa4=
 github.com/bgentry/speakeasy v0.1.0 h1:ByYyxL9InA1OWqxJqqp2A5pYHUrCiAL6K3J+LKSsQkY=
 github.com/bgentry/speakeasy v0.1.0/go.mod h1:+zsyZBPWlz7T6j88CTgSN5bM796AkVf0kBD4zp0CCIs=
-github.com/bketelsen/crypt v0.0.3-0.20200106085610-5cbc8cc4026c/go.mod h1:MKsuJmJgSg28kpZDP6UIiPt0e0Oz0kqKNGyRaWEPv84=
 github.com/blang/semver v3.5.0+incompatible/go.mod h1:kRBLl5iJ+tD4TcOOxsy/0fnwebNt5EWlYSAyrTnjyyk=
 github.com/blang/semver v3.5.1+incompatible h1:cQNTCjp13qL8KC3Nbxr/y2Bqb63oX6wdnnjpJbkM4JQ=
 github.com/blang/semver v3.5.1+incompatible/go.mod h1:kRBLl5iJ+tD4TcOOxsy/0fnwebNt5EWlYSAyrTnjyyk=
 github.com/boombuler/barcode v1.0.1-0.20190219062509-6c824513bacc/go.mod h1:paBWMcWSl3LHKBqUq+rly7CNSldXjb2rDl3JlRe0mD8=
 github.com/cenkalti/backoff v2.2.1+incompatible/go.mod h1:90ReRw6GdpyfrHakVjL/QHaoyV4aDUVVkXQJJJ3NXXM=
 github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA12rnyqOA5BBL4O983OfeGPqjHWSTneU=
-github.com/certifi/gocertifi v0.0.0-20200211180108-c7c1fbc02894/go.mod h1:sGbDF6GwGcLpkNXPUTkMRoywsNa/ol15pxFe6ERfguA=
 github.com/cespare/xxhash v1.1.0/go.mod h1:XrSqR1VqqWfGrhpAt58auRo0WTKS1nRRg3ghfAqPWnc=
-github.com/chanzuckerberg/aws-oidc v0.23.1/go.mod h1:AbN6f7tGQ1aR8kJdr5Mpnyk3iabBONFFLlAVQG8UgqM=
-github.com/chanzuckerberg/go-misc v0.0.0-20200814145037-61fd36eedee6/go.mod h1:5GljuH2+xw/T7ovs4sRj9WbNRrgePgom8sMzJJPlAAo=
 github.com/chanzuckerberg/go-misc v0.0.0-20201208181439-ea3e3e39e194 h1:Te8+Bfs5/1LQ9v0r8h7ocrCR7WXGp+Y61dbQhPIVtuc=
 github.com/chanzuckerberg/go-misc v0.0.0-20201208181439-ea3e3e39e194/go.mod h1:I+XR+Hhcty6cSO2ZBXNmlvxeGOr6yhE+JfEDQyOFtv4=
-github.com/chanzuckerberg/go-misc v0.0.0-20210727205931-6daf95f6727e h1:LyRaq/Sz1omD+bBuVwZBVxAv3qEUHAzgol5IFdG1YtY=
-github.com/chanzuckerberg/go-misc v0.0.0-20210727205931-6daf95f6727e/go.mod h1:mQwH9ExhxX/18odRhummSPc21vJ7TqB5y8u4uYv+1h4=
+github.com/cheggaaa/pb v1.0.27/go.mod h1:pQciLPpbU0oxA0h+VJYYLxO+XeDQb5pZijXscXHm81s=
```

Comment:
@alldoami Is you run `go mod graph` you could see that `go-misc v0.0.0-20210727205931-6daf95f6727e` was added the browser depedency that is breaking the release?

Reply:
Could you explain how this works for future reference 😅

Reply:
@alldoami I also don't know. All I know is that when the go mods where updated the go-misc mod of chanzuckerberg was also updated and in the new updated version it's pulling a new version of the indirect dependency browser pkg which broke the release.  Using `go mod graph` you can find out where the indirect dependency is coming from, which in this case was flgo-misc